com.unity.perception/Runtime/Randomization/Randomizers/RandomizerExamples/Randomizers/ConstantTransformRandomizer.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using UnityEngine.Experimental.Perception.Randomization.Randomizers.SampleRandomizers.Tags;
+
+namespace UnityEngine.Experimental.Perception.Randomization.Randomizers.SampleRandomizers
+{
+    /// <summary>
+    /// Sets a gameobject tagged with <see cref="ConstantTransformRandomizerTag"/> to the same transform(location, rotation, scale)
+    /// each frame. This randomizer is useful to reset a scene to a start state, especially when used in conjunction
+    /// with other randomizers that may move a gameobject.
+    /// </summary>
+    [Serializable]
+    [AddRandomizerMenu("Perception/Constant Transform Randomizer")]
+    public class ConstantTransformRandomizer : Randomizer
```

Comment:
I assume this is for the animation randomizer? On Nest we turned off root motion on the animations to address this (at least partially) https://docs.unity3d.com/Manual/RootMotion.html

Maybe we want to just add this feature of saving off the transform and resetting at next randomization to the animation randomizer? At the moment this randomizer feels like it will be a bit of a pain to use.

Reply:
I see you already have the root motion option! Makes sense

Reply:
Yeah, figured that out later. I think I'm going to remove this randomizer.

Reply:
I have removed this class